docs/backend/guides/dtod/testing-dtod.md

```diff
@@ -0,0 +1,51 @@
+# Testing DTOD
+
+The instructions that follow will say to change the environment variable inside of `.envrc`, however, there are other methods one can take to update to environment variables.
+
+- Environment variables can be changed inside of `.envrc.local`
+- Runing `export DTOD_USE_MOCK=false` after `direnv allow`
+
```

Comment:
**suggestion (non-blocking)**: A third option is to run the server like this:

```shell
export DTOD_USE_MOCK=false make server_run
```

Reply:
Oh wait, sorry, friday afternoon brain made a mistake, the export shouldn't be here, so just

```shell
DTOD_USE_MOCK=false make server_run
```

sorry about that 😞